test/integration-tests/gas-tests/V3ToV4Migration.gas.test.ts

```diff
@@ -566,6 +736,22 @@ describe('V3 to V4 Migration Gas Tests', () => {
 
         planner.addCommand(CommandType.V4_POSITION_MANAGER_CALL, [calldata])
 
+        compact = (
```

Comment:
i think it needs a comment on each of these un-permit things too to show whats happening to call increase.